packages/synapse-react-client/package.json

```diff
@@ -14,10 +14,8 @@
       "import": "./dist/*",
       "require": "./dist/*"
     },
-    "./package.json": {
-      "import": "./package.json",
-      "require": "./package.json"
-    }
+    "./package.json": "./package.json",
+    "./ThemeTypes": "./src/theme/ThemeTypes.d.ts"
```

Comment:
Because we changed our TSConfig `moduleResolution` to now be `bundler`, the apps in the monorepo now use this `exports` block to locate scripts to import into those individual projects.

For this reason, apps can no longer import by doing `synapse-react-client/src/...`. We were trying to prevent this from happening before by convention, but now our TypeScript config enforces it.

For our MUI type augmentations to work (such as our custom Typography variant strings), the apps have to import this `ThemesType` declaration file. This entry allows consumers to import `synapse-react-client/ThemeTypes`.

Note that this isn't portable outside of the monorepo, because the `src` folder isn't uploaded to pnpm, but this is not a new problem.